clients/java/src/test/java/io/camunda/zeebe/client/user/CreateUserTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.client.user;
+
+import static io.camunda.zeebe.client.impl.http.HttpClientFactory.REST_API_PATH;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.zeebe.client.api.command.ProblemException;
+import io.camunda.zeebe.client.protocol.rest.ProblemDetail;
+import io.camunda.zeebe.client.protocol.rest.UserWithPasswordRequest;
+import io.camunda.zeebe.client.util.ClientRestTest;
+import org.junit.jupiter.api.Test;
+
+public class CreateUserTest extends ClientRestTest {
+  @Test
+  void shouldCreateUser() {
+    // when
+    client
+        .newUserCreateCommand()
+        .username("username")
+        .name("name")
+        .email("email@example.com")
+        .password("password")
+        .send()
+        .join();
+
+    // then
+    final UserWithPasswordRequest request =
+        gatewayService.getLastRequest(UserWithPasswordRequest.class);
+    assertThat(request.getUsername()).isEqualTo("username");
+  }
+
+  @Test
+  void shouldRaiseExceptionOnNullUsername() {
```

Comment:
❌ The other fields are also required. Let's add test scenarios that verify these must be set as well, instead of just for the username.